dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/v2/controller/ProjectV2Controller.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.api.v2.controller;
+
+import static org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant.*;
```

Comment:
@abzymeinsjtu Wildcard imports are not allowed and that is the reason why CI failed again. However, `spotless` could not automatically fix them, actually this is the only style rule `spotless` could find out but cannot fix for you in dolphin. You need to manually remove them. To make sure wildcard imports are removed, you could run `mvn spotless:check` and make sure not errors pop up.